### PR TITLE
Incorporate useful link text.

### PR DESCRIPTION
**Pull request by osg-grafana**
Source content is at (internal doc) https://docs.google.com/document/d/1KODybKQA5lL5v5wRqcq4heO7Ie8MGt0M6aTKabtVcwI/edit.

### CODE REVIEW

**Comment by osg-grafana** [comment]
@AnneB-SEO2, there is no way to use placeholder URLs without the linter blocking the build from passing. So, I think that verbatim markup will have to do for now. We can always improve this later as well.

**Review by jdbaldry — APPROVED** [review]
LGTM assuming the copy has already been reviewed.

**Comment by osg-grafana** [comment]
> LGTM assuming the copy has already been reviewed.

Thank you! I pinged Anne to have a look at this first iteration before I merge this.

**Review by AnneB-SEO2 — APPROVED** [review]

**Pull request merged**
